app/assets/src/components/views/SampleView/SampleViewControls.jsx

```diff
@@ -10,6 +10,7 @@ import {
   getDownloadDropdownOptions,
   getLinkInfoForDownloadOption,
 } from "~/components/views/report/utils/download";
+import Nanobar from "nanobar";
```

Comment:
nit: can move to the top group for 3rd-party imports

Reply:
ok thx